### ISSUE TITLE
Parser breaks if there's space(s) between '$' sign and number

### ISSUE THREAD

**Issue by seltzered** [glasserc/ledger-to-beancount#2]
Noticed that if there's space(s) between the $ sign and the number for an entry, the parser crashes. 

Example:

```
2010-02-04 Purchasing new shoes
    Expenses:Clothing       $ 100
    Liabilities:Credit Card
```

results in the trace:
```
Traceback (most recent call last):
  File "/Users/hello/.pyenv/versions/3.5.0/bin/ledger-to-beancount", line 9, in <module>
    load_entry_point('ledger-to-beancount==0.0.1', 'console_scripts', 'ledger-to-beancount')()
  File "/Users/hello/.pyenv/versions/3.5.0/lib/python3.5/site-packages/ledger_to_beancount/__main__.py", line 7, in main
    output = translate_file(open(filename).readlines())
  File "/Users/hello/.pyenv/versions/3.5.0/lib/python3.5/site-packages/ledger_to_beancount/__init__.py", line 183, in translate_file
    posting = '  {}        {}'.format(account, translate_amount(rest))
  File "/Users/hello/.pyenv/versions/3.5.0/lib/python3.5/site-packages/ledger_to_beancount/__init__.py", line 89, in translate_amount
    amount = re.sub(r'\$' + amount_re, partial(replace_number, 'USD'), amount)
  File "/Users/hello/.pyenv/versions/3.5.0/lib/python3.5/re.py", line 182, in sub
    return _compile(pattern, flags).sub(repl, string, count)
  File "/Users/hello/.pyenv/versions/3.5.0/lib/python3.5/site-packages/ledger_to_beancount/__init__.py", line 87, in replace_number
    amount = Decimal(match.group(1))
decimal.InvalidOperation: [<class 'decimal.ConversionSyntax'>]
```

**Comment by seltzered** [comment]
appending `\s?` to the currency subs in `translate_amount` fixes things, will submit a pr.

**Comment by tbm** [comment]
This can be closed since it was fixed in 5b1a57b
